psiminer-core/src/test/kotlin/PsiUtils.kt

```diff
@@ -12,7 +12,7 @@ fun getAllMethods(file: File, handler: LanguageHandler, fixture: CodeInsightTest
         .let {
             ReadAction.compute<Map<String?, PsiElement>, Exception> {
                 handler
-                    .splitByGranularity(it, GranularityLevel.Method)
+                    .splitByGranularity(it as PsiElement, GranularityLevel.Method)
```

Comment:
Since PsiFile is a subclass of a PsiElement then it's unnecessary to make this cast.